HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -3950,4 +3951,32 @@ def self.cleanup_zones_spaces(hpxml_bldg)
     auto_zone = hpxml_bldg.conditioned_zones.find { |zone| zone.id.start_with? Constants.AutomaticallyAdded }
     auto_zone.delete if not auto_zone.nil?
   end
+
+  # Get the default weekday/weekend schedule fractions and monthly multipliers for each end use.
+  #
+  # @return [Hash] { schedule_name => { element => values, ... }, ... }
+  def self.get_default_schedules_csv_data()
+    default_schedules_csv = File.join(File.dirname(__FILE__), 'data', 'default_schedules.csv')
+    if not File.exist?(default_schedules_csv)
+      fail 'Could not find default_schedules.csv'
+    end
+
+    require 'csv'
+    default_schedules_csv_data = {}
+    default_schedules_data_sources = {}
+    CSV.foreach(default_schedules_csv, headers: true) do |row|
+      schedule_name = row['Schedule Name']
+      element = row['Element']
+      values = row['Values']
+      data_source = row['Data Source']
+
+      default_schedules_csv_data[schedule_name] = {} if !default_schedules_csv_data.keys.include?(schedule_name)
+      default_schedules_csv_data[schedule_name][element] = values
+
+      default_schedules_data_sources[schedule_name] = {} if !default_schedules_data_sources.keys.include?(schedule_name)
+      default_schedules_data_sources[schedule_name][element] = data_source
+    end
+
+    return default_schedules_csv_data, default_schedules_data_sources
```

Comment:
Looks like the docs script was the only user of the second return object, so I think it can be removed?